DRODLib/Briar.cpp

```diff
@@ -314,6 +315,15 @@ void CBriars::expand(
 							bBlocked = true;
 						break;
 
+						case M_CHARACTER: {
+							// Visible NPCs with Briar Immunity stop briar
+							const CCharacter *pCharacter = DYN_CAST(const CCharacter*, const CMonster*, pMonster);
+							if (!pCharacter) break;
+							if (!pCharacter->IsVisible()) break;
+							if (pCharacter->IsBriarImmune()) bBlocked = true;
```

Comment:
Please place the 'then' statement on the next line, so that a debug break point may be placed on it.